### ISSUE TITLE
How to report issues

### ISSUE THREAD

**Issue by etalaq** [wso2/product-sp#481]
Need to provide information and documentation on how to report issues about misbehavior of modules of the product.

For example, when defining a source for JSON file (tail based) to be read, some transactions result in error mentioning that the JSON format is not correct. But when providing the single record alone (using echo command for example) it works fine and the tool doesn't report the same error (for the same record).

So basically, the same record shows an error when processed along bulk of records but doesn't show an error when written along in a single format.

How to enable debug? How to provide the proper traces in order to identify the problem in order to help the developers?

**Comment by this** [comment]
Did you refer Siddhi docs (https://wso2.github.io/siddhi/) or SP docs https://docs.wso2.com/display/SP400/Stream+Processor+Documentation)?

**Comment by this** [comment]
Hope your questions have been answered by doc.
Closing due to inactivity.
